kong/plugins/key-auth/migrations/002_130_to_140.lua

```diff
@@ -27,6 +27,13 @@ return {
         -- Do nothing, accept existing state
       END$$;
 
+      DO $$
+      BEGIN
+        ALTER TABLE IF EXISTS ONLY "keyauth_credentials" ADD "ttl" TIMESTAMP WITH TIME ZONE;
+      EXCEPTION WHEN DUPLICATE_COLUMN THEN
+        -- Do nothing, accept existing state
+      END$$;
+
```

Comment:
Do we need to start adding indexes on `ttl` column? I see we don't have one for OAuth 2.0 either.

Reply:
Added index on `ttl` column on `keyauth_credentials`, `oauth2_authorization_codes` and `oauth2_tokens`